index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/c3c0d932d2.js" crossorigin="anonymous"></script>
+  <title>Document</title>
+</head>
+<body>
+  <header class="navbar">
+    <i class="fa-solid fa-bars menu-btn" id="menu-btn"></i>
+    <div class="navbar-menu">
+      <div class="menu-options">
+        <ul class="options">
+          <li class="nav-icon"><i class="fa-brands fa-facebook"></i></li>
+          <li class="nav-icon"><i class="fa-brands fa-square-twitter"></i></li>
+          <li class="nav-option">English</li>
+          <li class="nav-option">My page</li>
+          <li class="nav-option">Logout</li>
+        </ul>
+      </div>
+      <nav class="menu-links" id="menu">
+        <i class="fa-regular fa-circle-xmark close-btn menu-btn" id="close-btn"></i>
+        <img src='media/event-logo.png' alt='logo' class='nav-logo' id='nav-logo'>
+        <ul class="nav-links">
+          <li class="nav-link"><a href="#" id="about-link">About</a></li>
+          <li class="nav-link"><a href="#">Program</a></li>
+          <li class="nav-link"><a href="#">Join</a></li>
+          <li class="nav-link"><a href="#">Sponsor</a></li>
+          <li class="nav-link"><a href="#">News</a></li>
+          <li class="nav-link"><a href="#">YWCC Campaign</a></li>
+        </ul>
```

Comment:
- [ ] Kindly implement the Orange button to the left as it is in the given design, also implement the pressed button effect. If you look closeley you will see that the page on which the user is has a highlighted color to its corresponding link in the navigation.
#### Note that this applies fr the about page as well

Your design
![image](https://user-images.githubusercontent.com/81932478/203663427-aece643d-94d5-444b-ad6c-8270777fb6ba.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/203663453-5c9dfdc2-1b58-4aa1-be68-1c6a7f118ff9.png)